bravado/mapping/schema.py

```diff
@@ -10,6 +11,14 @@
     'null': (),
 }
 
+SWAGGER_PRIMITIVES = (
+    'integer',
+    'number',
+    'string',
+    'boolean',
+    'null'     # TODO: Do we need this?
```

Comment:
- Removed TODO for the null.
- Added TODOs for support `file` type on marshal/unmarshal